parsers/verilog.c

```diff
@@ -1577,8 +1580,26 @@ static int tagNameList (tokenInfo* token, int c)
 	{
 		c = processType(token, c, &kind);	// update token and kind
 
-		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`')
+		if (c == '=' || c == ',' || c == ';' || c == ')' || c == '`' || c == '(' || c == '[')
 		{
+			while(c == '[')
+			{
+				c = skipPastMatch ("[]");
+			}
+			if(c == '(')
+			{
+				c = skipPastMatch ("()");
```

Comment:
We need `c= skipWhite (c);` here.

`m inst ()  ; // a space before semicolon` fails.   (You may want to add a test case for this. )

`skipPastMatch()` should do the job, but it does not now.  I will work on this issue on this weekend.
You may wait for the fix.